composer.json

```diff
@@ -8,45 +8,26 @@
     "require": {
         "php": ">=8.0",
         "cweagans/composer-patches": "~1.4",
-        "drupal/core": "^9.4"
+        "drupal/core": "^9.4 || ^10"
     },
     "require-dev": {
-        "behat/behat": "^3.10",
-        "behat/mink-extension": "^2.3.1",
-        "behat/mink-selenium2-driver": "^1.5",
         "composer/installers": "^1.5",
         "drupal/address": "^1.8",
-        "drupal/config_devel": "^1.2",
+        "drupal/config_devel": "^1.9",
         "drupal/content_lock": "^2.2",
-        "drupal/core-composer-scaffold": "^9.4",
-        "drupal/ctools": "^3.9.0",
-        "drupal/drupal-extension": "dev-master as 4.0",
+        "drupal/core-composer-scaffold": "^9.4 || ^10",
+        "drupal/drupal-extension": "^5.0",
         "drupal/entity_version": "^1.1",
-        "drupal/token": "^1.10",
         "drush/drush": "^11.1",
-        "guzzlehttp/guzzle": "^7.0",
-        "instaclick/php-webdriver": "^1.4.7",
-        "masterminds/html5": "^2.7.6",
         "mikey179/vfsstream": "^1.6.10",
-        "nikic/php-parser": "^4.12.0",
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "^2.0",
-        "openeuropa/oe_multilingual": "^1.12",
-        "openeuropa/oe_paragraphs": "^1.12",
-        "openeuropa/oe_translation": "^0.18.1",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
+        "openeuropa/oe_multilingual": "^1.13",
+        "openeuropa/oe_paragraphs": "^1.19",
+        "openeuropa/oe_translation": "dev-EWPP-2965-2.x",
```

Comment:
So, normally we moved the editorial integration to oe_trans. So this dependnecy and submodule can be removed entirely.